src/react-apps/ux-editor/package.json

```diff
@@ -18,14 +18,16 @@
   "author": "Altinn",
   "license": "MIT",
   "dependencies": {
+    "@material-ui/core": "^3.4.0",
     "axios": "0.18.0",
     "babel-core": "6.26.3",
+    "classnames": "^2.2.6",
     "deepmerge": "2.1.1",
     "dot-object": "^1.7.0",
     "immutability-helper": "2.7.0",
     "lodash.isequal": "^4.5.0",
-    "react": "16.4.0",
-    "react-dom": "16.4.0",
+    "react": "16.5.2",
```

Comment:
Weren't we going to use 16.6.x of react?

Reply:
Yes. I think that was technically part of another issue, but since we needed a newer version here I have upgraded to v16.6.1 now.